db/db.py

```diff
@@ -271,27 +281,78 @@ class Job(Base):
     timeout_seconds = Column(Integer, nullable=False)
     email_notifications = Column(JSON, nullable=False)
     workspace_id = Column(String, ForeignKey("workspaces.id"), nullable=False)
-    workspace = relationship(Workspace)
     new_cluster = Column(JSON)
+    existing_cluster_id = Column(String)
     schedule_quartz_cron_expression = Column(String)
     schedule_timezone_id = Column(String)
     task_type = Column(String)
-    notebook_path = Column(String)
-    notebook_revision_timestamp = Column(Integer)
+    task_parameters = Column(JSON)
     max_concurrent_runs = Column(Integer)
 
+    workspace = relationship(Workspace)
+    jobruns = relationship("JobRun",
+                           back_populates='job',
+                           order_by="JobRun.start_time")
+
+    def runs(self, last=None, since_days=None):
```

Comment:
`runs` is not very descriptive either. Maybe adding documentation would help it.

Reply:
I think this has a proper name, it'll returns the jobruns. Although I'll add docstring to describe it's purpose.